README.md

```diff
@@ -434,20 +434,26 @@ users
 > In all the cases, the `User ID` parameter is the unique identifier of the auth0 account instance. i.e. in `google-oauth2|123456789081523216417` it would be the part after the '|' pipe: `123456789081523216417`.
 
 
-### Credentials Manager
-This library ships with a `CredentialsManager` class to easily store and retrieve fresh Credentials from a given `Storage`.
```

Comment:
Personally I don't like implying levels of difficulty in usage.

~class to easily store and retrieve fresh Credentials from a given~ -> utility class to manage the store, retrieve and renew credentials from a given `Storage`.

Do we need to specify from a given storage?

Reply:
🤔  I think you're commenting on the deleted line.